app/routes/admin/projects/index.js

```diff
@@ -0,0 +1,7 @@
+import Route from '@ember/routing/route';
+
+export default Route.extend({
+  model() {
+    return this.store.findAll('project');
```

Comment:
This is fine for now, though we'll need to eventually paginate it.

Reply:
Probably, same as with org invites, albeit probably sooner than that.